Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -99,7 +99,9 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
     let scaledLineThickness = lineThickness;
     if (publicAPI.getScaleInPixels()) {
       const centerCoords = model.pipelines.center.source.getCenter();
-      scaledLineThickness *= publicAPI.getPixelWorldHeightAtCoord(centerCoords);
+      scaledLineThickness *=
+        window.devicePixelRatio *
```

Comment:
@floryst how about adding devicePixelRatio in `getPixelWorldHeightAtCoord` ?

Reply:
I think it would be nice to thread devicePixelRatio through here for more control over it, even though its not terribly useful from a functional standpoint.

Reply:
Just to make sure I understand your sentence, are you suggesting to apply `devicePixelRatio` in `getPixelWorldHeightAtCoord()` or not ?

Reply:
@floryst can you please precise what you meant ?

Reply:
Sorry, you can ignore what I said before. I think I was mis-interpreting your question.

Yes, I think it's a good idea to do the `window.devicePixelRatio` multiply within `getPixelWorldHeightAtCoord.